src/index.html

```diff
@@ -1,12 +1,430 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page-content">
+    <header class="header page-content__offset">
+      <a href="#">
+        <img
+        src="images/logo.png"
+        alt="logo"
+        class="header__logo"
+      >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="apple"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="samsung"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="phones"
+              class="header__nav-link"
+            >
+              Смартфоны и Телефоны
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="laptops"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="gadgets"
+              class="header__nav-link"
+            >
+              Гаджеты
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="tablets"
+              class="header__nav-link"
+            >
+              Планшеты
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="photo"
+              class="header__nav-link"
+            >
+              Фото
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="video"
+              class="header__nav-link"
+            >
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__wrapper catalog">
+        <section class="product-card catalog__card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="image"
+            class="product-card__image"
+          >
+          <div class="product-info">
+            <div class="product-info__name">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+            <p class="product-info__code">
+              Код товара: 195434
+            </p>
+          </div>
+          <div class="product-rating">
```

Comment:
You are making card structure overcomplicated.
Let's leave one block - `product-card` and make all inner items its elements.
Only exception - `stars` block that we reuse from previous task.

Reply:
If my memory serves me right, the card has been reused from the last task approved by you.

Reply:
It is not a mistake, it is just advice on better code structuring.
You can write it this way if it's convenient for you.

Reply:
> It is not a mistake, it is just advice on better code structuring.
> You can write it this way if it's convenient for you.

Ok, thanks:) Will adhere to your advices.